src/tests/subscriptions.test.js

```diff
@@ -1,3 +1,153 @@
-test('should pass a basic test', () => {
-  expect(1 + 1).toBe(2);
+import request from 'supertest';
+import app from '../app';
+import Subscription from '../models/Subscription';
+import Member from '../models/Member';
+import Classes from '../models/Class';
+import subscriptionSeed from '../seeds/subscriptions';
+import classSeed from '../seeds/classes';
+import memberSeed from '../seeds/members';
+
+const newSubscription = {
+  _id: '6466aeb229ef0fba19e2cd85',
+  classes: '6465816c08ae1e4d4b105a49',
+  member: '646223b3731da1b875752bda',
+};
+
+const duplicatedId = {
+  _id: '6466aeb229ef0fba19e2cd85',
+  classes: '6465816c08ae1e4d4b105a49',
+  member: '646223b3731da1b875752bda',
+};
+
+const dataBase = {
+  classes: '64663d50bb2d87b9f6510625',
+  member: '24625faf956fb774c56da140',
+};
+
+const noValidData = {
+  classes: '---64663d50bb2d87b9f6510625',
+  member: '--24625faf956fb774c56da140',
+};
+
+const dataSubscription = {
+  classes: '74663d50bb2d87b9f6510624',
+  member: '6462d0074441252c694332dd',
+  date: new Date(),
+};
+
+beforeAll(async () => {
+  await Subscription.collection.insertMany(subscriptionSeed);
+  await Classes.collection.insertMany(classSeed);
+  await Member.collection.insertMany(memberSeed);
+});
+
+describe('GET /api/subscriptions', () => {
+  test('if data isnt empty should return 200 and its array', async () => {
+    const response = await request(app).get('/api/subscriptions').send();
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
```

Comment:
Try to make a test with a server error. You can try to create a test where the api path is incorrect.